docs/uax-with-aj1/Request-to-revise-UAX-50-for-the-harmonization-with-Adobe-Japan1.md

```diff
@@ -0,0 +1,55 @@
+### Contributors:
+* Yasuo Kida (independent)
+* Taro Yamamoto (Adobe)
+* Nat McCully (Adobe)
+* Kobayashi Toshi (independent)
+* KOBAYASHI Tatsuo (independent)
+* Atsushi Shimono (W3C)
+* Shinyu MURAKAMI (Vivliostyle)
+* MURATA Makoto (Keio University)
+
+***
+
+[CITPC(Council for the Promotion of Character Information Technology)](http://citpc.jp) is a general incorporated association in Japan.  CITPC aims to enhance the interoperability of character information processing in Japan.  Most of the font vendors in Japan are members of CITPC.  
+
+UAX 50 was designed as a cornerstone of vertical writing for the Web and EPUB.  CITPC is grateful to the Unicode Consortium for the timely development of UAX 50 and its ongoing maintenance.   
+
+However, there are some inconsistencies between UAX 50 and the Adobe Japan1 character collection (as defined in a GitHub repository, available at [https://github.com/adobe-type-tools/Adobe-Japan1](https://github.com/adobe-type-tools/Adobe-Japan1)).  Since many important fonts in Japan are based on the Adobe Japan1 character collection, impacts of these inconsistencies should not be underestimated.

```

Comment:
```suggestion
However, there are some inconsistencies between UAX 50 and the Adobe Japan1 character collection (as defined in a GitHub repository, available at [https://github.com/adobe-type-tools/Adobe-Japan1](https://github.com/adobe-type-tools/Adobe-Japan1)).  Since many important fonts in Japan are based on the Adobe Japan1 character collection, the impacts of these inconsistencies should not be underestimated.

```

Reply:
it's plural form, and not sure we should add 'the' here....

Reply:
I'll let @r12a decide whether "the" is needed here.